frontend/components/Editor/Editor.tsx

```diff
@@ -8,7 +8,7 @@ const baseClass = "editor";
 interface IEditorProps {
   focus?: boolean;
   label?: string;
-  labelTooltip?: string;
+  labelTooltip?: string | JSX.Element;
```

Comment:
We were moving towards using `React.ReactNode` to encompass both of these right?

Reply:
Initially I had changed it `ReactNode` but went with `string | JSX.Element` since I haven't fully internalized the difference between this and that and didn't want to introduce subtle issues with a potentially broader type.